rllib/agents/trainer.py

```diff
@@ -578,6 +561,7 @@ def normalize(env):
             self.env_creator = lambda env_config: normalize(inner(env_config))
 
         Trainer._validate_config(self.config)
+        self.callbacks = self.config["callbacks"]()
```

Comment:
consider doing a type check here to provide a better user error

Reply:
Done